.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on: 
+    push:
+        branches:
+            - master
+    pull_request:
+
```

Comment:
if you add to pull_request `branches: -master` it should run GA when creating a PR